e2e/tests/nightly/gke-apps/cromwell.spec.ts

```diff
@@ -60,7 +66,10 @@ describe('Cromwell GKE App', () => {
     await workspaceDataPage.deleteWorkspace();
   });
 
-  test('Run cromwell using python notebook', async () => {
+  test.each([
+    [Language.Python, 'All of Us Cromwell Setup Python snippets', cromshellSubmitPythonCmd],
+    [Language.R, 'All of Us Cromwell Setup snippets', cromshellSubmitRCmd]
+  ])('Run cromwell using notebook', async (language, snippetMenu, cromshellSubmitCommand) => {
```

Comment:
The test name should include the programming language so we can tell which language fails the test when this runs.